data/map.txt

```diff
@@ -31215,6 +32942,19 @@ planet "Sasirka Gatru"
 	landscape land/desert7
 	description `This was probably a habitable world at one point. It is no longer. Nothing lives or grows here, and the atmosphere is full of toxic chemicals.`
 
+planet Schisathro
+	attributes bunrodea guild manufacturing mining
+	landscape land/canyon5
+	description `Schisathro is a major mining world, and also home to the largest shipyards in Bunrodean space. Larger shipyards than the ones found here only exist on the capital world of Erabuthro.`
```

Comment:
```suggestion
	description `Schisathro is a major mining world, and also home to some of the largest shipyards in Bunrodean space. Larger shipyards than the ones found here only exist on the capital world of Erabuthro.`
```

Reply:
No, these *are* the largest.

Reply:
They can't be the largest, because you're saying that larger ones exist on Erabuthro.

Reply:
True.